### PR TITLE
Add all properties to GeoJsonLayer, removePolygons method.

### PR DESCRIPTION
**Pull request by EBartusiak**
- Added remaining properties to GeoJsonLayer.
- Added removePolygons method, will add removePolylines and removeIcons in a subsequent commit.
- Changes to GeoJsonParser can be ignored (only change was formatting).

### CODE REVIEW

File: iOS/MSMapsModules/MSMapsModules/MSMapGeoJsonParser/MSMapGeoJsonLayer.m [hunk 1]
```diff
@@ -11,39 +11,129 @@
...
-	if (_fillColor != newfillColor) {
-		_fillColor = newfillColor;
-		for (MSMapElement *element in [super elements]) {
-			if ([element isKindOfClass:[MSMapPolygon class]]) {
-				MSMapPolygon *polygon = (MSMapPolygon *)element;
-				polygon.fillColor = _fillColor;
-			}
-		}
-	}
+  if (_fillColor != newfillColor) {
```
Comment (KristofferHenriksson) [inline iOS/MSMapsModules/MSMapsModules/MSMapGeoJsonParser/MSMapGeoJsonLayer.m:29]: Can you make a separate pull request for just the formatting changes and then make the code changes in this one? It's more difficult to review when code and formatting changes are intermingled.

Comment (EBartusiak) [reply iOS/MSMapsModules/MSMapsModules/MSMapGeoJsonParser/MSMapGeoJsonLayer.m:29]: New PR created for clarity.
